src/components/Calculator.jsx

```diff
@@ -11,6 +11,7 @@ const Calculator = () => {
   return (
     <div>
       <div className="calculator">
+        <h1>Let&apos;s do some math!</h1>
         <div className="screen">
           <Screen result={result.next || result.total || '0'} />
         </div>
```

Comment:
- [ ] For the layout of the page, you're expected to display the Let's do some math! text on the left and the Calculator on the right just as it is in the wireframe. Kindly work on your styles to make that possible 👍🏽

<img width="438" alt="Screenshot 2023-04-03 at 3 41 12 PM" src="https://user-images.githubusercontent.com/50941074/229559696-e306911e-b8e2-4c98-841a-ee772eb6aebf.png">